### PR TITLE
Authors page list with search

### PR DESCRIPTION
**Pull request by vzanimonets**
- Implementation of director's page list in src/pages/architects.js
- search director by  first name & second name

### CODE REVIEW

File: src/components/SearchBar.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react'
+import { Input } from '@material-ui/core'
+
+const inputStyle = {
```
Comment (vadzim-sasonkin) [inline src/components/SearchBar.js]: If it is not overriding of common input styles it's better to use material theme values or just configure input on theme creation

File: src/components/SearchBar.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  color: '#000',
+  border: '1px solid #000',
+  width: '100%',
+  boxShadow: 'none'
+}
+const SearchBar = ({ sortFunc }) => {
+  const [value, setValue] = useState('')
+  const handleSubmit = e => {
+    e.preventDefault()
```
Comment (vadzim-sasonkin) [inline src/components/SearchBar.js]: What is the purpose of preventing default here?

File: src/components/SearchBar.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        className="input-search"
+        label="Search Director"
+        type="search"
+        onChange={e => {
+          handleSubmit(e)
+        }}
+        style={inputStyle}
+      />
+
+      {/* <SearchBar
```
Comment (vadzim-sasonkin) [inline src/components/SearchBar.js]: Try to avoid unhelpful comments

File: src/components/DirectorListItem.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  root: {
+    ' & a': { width: '100%' }
+  }
+}))
+
+const DirectorListItem = ({ directorData }) => {
+  const classes = useStyles()
+  const [selectedIndex, setSelectedIndex] = React.useState(1)
+
+  const handleListItemClick = (event, index) => {
```
Comment (vadzim-sasonkin) [inline src/components/DirectorListItem.js]: I am a little bit confused with index here, why it's always zero and why is it initialized in the state with 1.

File: src/pages/architects.js [hunk 5]
```diff
@@ -1,16 +1,59 @@
 import React from 'react'
```
Comment (vadzim-sasonkin) [inline src/pages/architects.js:1]: So we can delete the architects page and leave only `directors.js` ?

File: src/pages/directors.js [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+          }
+        }
+      }
+    }
+  `)
+  const [directorsArr] = data.allDirectorsJson.nodes.map(node => node.directors)
+
+  const [searchResults, setSearchResults] = React.useState(directorsArr)
+
+  const filterDirectors = value => {
```
Comment (vadzim-sasonkin) [inline src/pages/directors.js:26]: Maybe it's better to take out comparator functionality from component - it won't be recreated in every render and the component would be more concise

**Review by vadzim-sasonkin — COMMENTED** [review]
Also, have a look at the localization feature, it was already merged to master.

File: src/components/DirectorListItem.js (hunk 4 above)
Comment (vzanimonets) [reply src/components/DirectorListItem.js]: it should be for future features, something like drag and drop, etc. It change it's value in onClick fn. Ok, I'm deleting it now.)

File: src/components/SearchBar.js (hunk 3 above)
Comment (vzanimonets) [reply src/components/SearchBar.js]: Done

File: src/components/SearchBar.js (hunk 2 above)
Comment (vzanimonets) [reply src/components/SearchBar.js]: deleted

File: src/components/SearchBar.js (hunk 1 above)
Comment (vzanimonets) [reply src/components/SearchBar.js]: I can remove it, but now default inputs don't have borders & have white color.

File: src/pages/architects.js (hunk 5 above)
Comment (vzanimonets) [reply src/pages/architects.js:1]: Yes we can, after commit fixes according code review

File: src/pages/directors.js (hunk 6 above)
Comment (vzanimonets) [reply src/pages/directors.js:26]: Yes, you are right! Done

File: src/components/SearchBar.js (hunk 1 above)
Comment (vadzim-sasonkin) [reply src/components/SearchBar.js]: I mean - if any input is assumed to have these styles, I'd rather apply them in global config like
https://material-ui.com/customization/globals/
If this input is unique and exists only inside Search, then it's ok.

**Review by vadzim-sasonkin — APPROVED** [review]

**Pull request merged**